compositeviews/compositeview01/composite_view.json

```diff
@@ -13,7 +13,7 @@
     {
       "@id": "connectome-sparql-projection-01",
       "@type": "SparqlProjection",
-      "query": "PREFIX nxv: <https://bluebrain.github.io/nexus/vocabulary/> PREFIX xsd: <http://www.w3.org/2001/XMLSchema#> CONSTRUCT { {resource_id} ?p ?o . } WHERE { {resource_id} ?p ?o . FILTER(!strstarts(str(?p),str(nxv:)) && ?o != \"validate\"^^xsd:string && ?o != \"validate@validate.com\"^^xsd:string && ?o != <http://schema.org/Thing>) }",
```

Comment:
@kaij not sure if we would have to add this in the ES projections as well. I think for the sake of completeness we should.

Reply:
Ok, we'll check agin